apps/api/src/auth/auth.service.ts

```diff
@@ -145,6 +145,22 @@ export class AuthService {
       if (!grant.access_token?.token) {
         throw new InternalServerErrorException('CannotIssueTokenError')
       }
+      const user = await this.keycloak.grantManager.userInfo<string, KeycloakTokenParsed>(
+        grant.access_token.token as string,
+      )
+      const person = await this.prismaService.person.findUnique({ where: { email: user.email } })
+      if (!person || person.keycloakId !== user.sub) {
+        Logger.warn('No person found for the current keycloak user. Creating new one...')
+        await this.authenticateAdmin()
```

Comment:
I assume we need authenticate as admin for the "createPerson" call.
Does this leave the user with admin rights after the login?

Reply:
No. This is needed to check, whether the server itself, can access the keycloak realm, with the credentials in the .env